src/main/webapp/app/guided-tour/guided-tour.component.html

```diff
@@ -37,6 +37,14 @@ <h5 jhiTranslate="{{ currentTourStep.subHeadlineTranslateKey }}" class="sub-head
                         <div [innerHTML]="currentTourStep.hintTranslateKey | translate | htmlForGuidedTourMarkdown"></div>
                     </div>
                 </div>
+                <div *ngIf="currentTourStep.alreadyExecutedTranslateKey" class="step-hint interaction alert alert-success">
```

Comment:
I would refactor all the hints into a new `guided-tour-hints` component in a future PR